docs/basic-features/script.md

```diff
@@ -69,33 +69,48 @@ function Home() {
 
 ```js
 import Script from 'next/script'
-;<Script
-  src="https://polyfill.io/v3/polyfill.min.js?features=IntersectionObserverEntry%2CIntersectionObserver"
-  strategy="beforeInteractive"
-/>
+
+const Home = () => (
```

Comment:
```suggestion
const Home = () => (
```

In general all docs use `function` to ensure it matches up with general recommendations (e.g. React docs). Can you change these to `function Home () {` instead. It'd be ideal to add `export default` as well given that it's a component assumed to be a page